### PR TITLE
Hometask2

### PR DESCRIPTION
**Pull request by sergbondarenko**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import UserTable from './components/UserTable'
+import AddUserForm from './components/AddUserForm'
+import { UserData } from './types';
+
+
+function App() {
+  const [usersList, SetUsersList] = React.useState<Array<UserData> >([]);
```
Comment (pannkotsky) [inline src/App.tsx:8]:
```suggestion
  const [usersList, setUsersList] = React.useState<Array<UserData> >([]);
```

File: src/App.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    return {
+      id: timestamp,
+      lastname: user.lastname,
+      name: user.name,
+      surname: user.surname
+    }
+  }
+  return (
+      <div>
+          <AddUserForm onSubmit={newUser => SetUsersList(oldUsersList => [...oldUsersList, newUser] )} />
```
Comment (pannkotsky) [inline src/App.tsx:20]:
```suggestion
          <AddUserForm onSubmit={newUser => setUsersList(oldUsersList => [...oldUsersList, newUser] )} />
```

File: src/components/AddUserForm.tsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    const [lastNameValue, setLastNameValue] = React.useState('');
+
+    return (
+        <div>
+            <form
+                autoComplete="off"
+                onSubmit={event => {
+                    event.preventDefault();
+                    onSubmit({
+                        id: 0,
```
Comment (pannkotsky) [inline src/components/AddUserForm.tsx:24]: you need to use unique ids for each user, or example timestamp or uuid

File: src/components/NameForm.tsx [hunk 4]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
```
Comment (pannkotsky) [inline src/components/NameForm.tsx:1]: this component seems not used, can be deleted

File: src/types.ts [hunk 5]
```diff
@@ -0,0 +1,12 @@
+export type Visit = {
+    id: number;
+    timestamp: number;
+    name: string;
+}
```
Comment (pannkotsky) [inline src/types.ts:5]: this type can be deleted

**Review by pannkotsky — APPROVED** [review]
Approving, but address my comments please

File: src/App.tsx (hunk 1 above)
Comment (sergbondarenko) [reply src/App.tsx:8]: Done

File: src/App.tsx (hunk 2 above)
Comment (sergbondarenko) [reply src/App.tsx:20]: Done

File: src/components/AddUserForm.tsx (hunk 3 above)
Comment (sergbondarenko) [reply src/components/AddUserForm.tsx:24]: Done

File: src/components/NameForm.tsx (hunk 4 above)
Comment (sergbondarenko) [reply src/components/NameForm.tsx:1]: Done

File: src/types.ts (hunk 5 above)
Comment (sergbondarenko) [reply src/types.ts:5]: Done

**Pull request merged**